updatecli/updatecli.d/inbound-agent.yml

```
@@ -0,0 +1,59 @@
+{{ $suffix := "-jdk21-nanoserver-1809" }}
+---
+name: Bump `jenkins/inbound-agent` Docker image version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  jenkins/inbound-agent:
+    name: '[jenkins/inbound-agent] Get latest GitHub release'
+    kind: githubrelease
+    spec:
+      owner: "jenkinsci"
+      repository: "docker-agent"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: regex
+        pattern: ^\d+\.v[a-f0-9_]+-\d+$
```

```suggestion
    transformers:
      - addsuffix: -jdk21-nanoserver-1809
```

- Removing versionFilter: is is unneeded as the "githubrelease" resource returns the GH release marked as "latest". Adding a regex is not sustainable as it assumes the version scheme will always be the same (and throwing regex at problems increases problems scope :p )

- Let's do the transformation at the source once for all. Simplifies usages as it avoids using go templating (which is tedious to debug and escape)